src/service/network_tables.h

```diff
@@ -49,100 +52,176 @@ namespace ccf
+    SnpHostDataMap host_data = {Tables::HOST_DATA};
+    SnpMeasurements snp_measurements = {Tables::NODE_SNP_MEASUREMENTS};
+
+    inline auto get_all_node_tables() const
+    {
+      return std::make_tuple(
+        node_code_ids,
+        nodes,
+        node_endorsed_certificates,
+        acme_certificates,
+        host_data,
+        snp_measurements);
+    }
+
+    //
+    // History of governance, proposals, and ballots tables
+    //
+    GovernanceHistory governance_history = {Tables::GOV_HISTORY};
+    COSEGovernanceHistory cose_governance_history = {Tables::COSE_GOV_HISTORY};
+    jsgov::ProposalMap proposals = {jsgov::Tables::PROPOSALS};
+    jsgov::ProposalInfoMap proposal_info = {jsgov::Tables::PROPOSALS_INFO};
+
+    inline auto get_all_governance_history_tables() const
+    {
+      return std::make_tuple(
+        governance_history, cose_governance_history, proposals, proposal_info);
+    }
+
+    //
+    // JS Generic tables
+    //
+    Modules modules = {Tables::MODULES};
+    ModulesQuickJsBytecode modules_quickjs_bytecode = {
+      Tables::MODULES_QUICKJS_BYTECODE};
+    ModulesQuickJsVersion modules_quickjs_version = {
+      Tables::MODULES_QUICKJS_VERSION};
+    JSEngine js_engine = {Tables::JSENGINE};
 
+    inline auto get_all_js_generic_tables() const
+    {
+      return std::make_tuple(
+        modules, modules_quickjs_bytecode, modules_quickjs_version, js_engine);
+    }
+
+    //
+    // JWT tables
+    //
+    CACertBundlePEMs ca_cert_bundles = {Tables::CA_CERT_BUNDLE_PEMS};
+    JwtIssuers jwt_issuers = {Tables::JWT_ISSUERS};
+    JwtPublicSigningKeys jwt_public_signing_keys = {
+      Tables::JWT_PUBLIC_SIGNING_KEYS};
+    JwtPublicSigningKeyIssuer jwt_public_signing_key_issuer = {
+      Tables::JWT_PUBLIC_SIGNING_KEY_ISSUER};
+
+    inline auto get_all_jwt_tables() const
+    {
+      return std::make_tuple(
+        ca_cert_bundles,
+        jwt_issuers,
+        jwt_public_signing_keys,
+        jwt_public_signing_key_issuer);
+    }
+
+    //
+    // Service tables
     //
-    // Internal CCF tables
+    Service service = {Tables::SERVICE};
+    PreviousServiceIdentity previous_service_identity = {
+      Tables::PREVIOUS_SERVICE_IDENTITY};
+    Configuration config = {Tables::CONFIGURATION};
+    Constitution constitution = {Tables::CONSTITUTION};
+
+    inline auto get_all_service_tables() const
+    {
+      return std::make_tuple(
+        service, config, constitution, previous_service_identity);
+    }
+
+    // All builtin governance tables should be included here, so that wrapper
+    // endpoints can be automatically generated for them
+    inline auto get_all_builtin_governance_tables() const
+    {
+      return std::tuple_cat(
+        get_all_member_tables(),
+        get_all_user_tables(),
+        get_all_node_tables(),
+        get_all_governance_history_tables(),
+        get_all_js_generic_tables(),
+        get_all_jwt_tables(),
+        get_all_service_tables());
+    }
```

Comment:
I've been saying for ages that `network_tables.h` is unnecessary, and we shouldn't use it. But now we have a use case for it - a single location to list all of the "builtin" governance tables. This is roughly all of the ones that C++ knows about/thinks exists, that we want to auto-generate wrappers for.

The other change in this file is to move the initialization up, so that the type and name are together rather than annoyingly separated.